code/calibration.h

```diff
@@ -1,11 +1,14 @@
 #ifndef CALIBRATION_H
+#include <stdint.h>
 
-const float BSGMaxVoltage = 55;
+const uint8_t BSGMaxVoltage;
 
-const uint16_t EngineMaxRPM = 10000;
-const uint16_t EngineMinOperationRPM = 1000;
+const uint16_t EngineMaxRPM;
+const uint16_t EngineMinOperationRPM;
 
-const float BatteryMinVoltage = 10;
-const float BatteryMaxVoltage = 60;
+const uint8_t BatteryMinVoltage;
+const uint8_t BatteryMinOperationVoltage;
+const uint8_t BatteryMaxVoltage;
+const uint8_t BatterySpecVoltage;
```

Comment:
Suggestion: you can fix the error in the CI by adding `extern` to each variable, as presented below:

```c
extern const uint8_t BSGMaxVoltage;

extern const uint16_t EngineMaxRPM;
extern const uint16_t EngineMinOperationRPM;

extern const uint8_t BatteryMinVoltage;
extern const uint8_t BatteryMinOperationVoltage;
extern const uint8_t BatteryMaxVoltage;
extern const uint8_t BatterySpecVoltage;
```